index.html

```diff
@@ -8,29 +8,161 @@
+                <button class="seeproject" type="button">See Project</button>
+            </div>
+        </div>
+        <div class="card1">
+            <img src="icons/Snapshoot Portfolio (3).png" alt="Snapshoot.png">
+            <div class="hero1">
+                <h2 class="Tonic">Multy Post <br>Stories</h2>
+                <div class="canopy">
+                    <span class="canopi">CANOPY</span>
+                    <img src="icons/Counter.png" alt="Counter.png">
+                    <span class="backend">Back End Dev</span>
+                    <img src="icons/Counter.png" alt="Counter.png">
+                    <span class="fiftin">2015</span>
+                </div>
+                <article class="art">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </article>
+                <ul class="frontend">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="seeproject" type="button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="aboutmyself">
+            <h2 class="heading">About<br> Myself</h2>
+            <article class="main">
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </article>
+            <h2 class="connect">LET'S CONNECT</h2>
+            <div>
+                <ul class="social">
+                    <li><a href="#"><img src="icons/Vector.png" alt="twitter_icon"></a></li>
+                    <li><a href="#"><img src="icons/Linkedin icon.png" alt="Linkedin_icon"></a></li>
+                    <li><a href="#"><img src="icons/Vector (1).png" alt="Medium_icon"></a></li>
+                    <li><a href="#"><img src="icons/Vector (2).png" alt="Github_icon"></a></li>
+                    <li><a href="#"><img src="icons/Vector (3).png" alt="Chill_icon"></a></li>
+                </ul>
+            </div> 
+            <button type="button" class="seeproject">Get my resume</button>   
+        </div>
+        <div id="about">
+            <ul class="languages">
+                <li><h2>Languages</h2></li>
+                <li><img src="icons/un.png" alt="un.png"></li>
             </ul>
-        </div>    
+            <ul class="prlanguages">
+                <li><img src="icons/Ellipse 1.png" alt="Ellipse">Javascript</li>
+                <li><img src="icons/Ellipse 1 (1).png" alt="Ellipse">HTML</li>
+                <li><img src="icons/Ellipse 1 (2).png" alt="Ellipse">CSS</li>
+            </ul>
+            <ul class="languages">
+                <li><h2>Frameworks</h2></li>
+                <li><img src="icons/u.png" alt="un.png"></li>
```

Comment:
- [optional] kindly reduce the gap between frameworks and skills, so it matches the Figma design. The font size as well to 20px like in the original design. Nice work
yours
![Screenshot (798)](https://user-images.githubusercontent.com/65298048/219591787-f599a045-d211-4a1c-90a6-f06a2149b024.png)
figma
![Screenshot (799)](https://user-images.githubusercontent.com/65298048/219591896-ab24524e-2ca2-48c8-9d64-758824b2184d.png)